test/kiba/plus/destination/mysql_test.rb

```diff
@@ -0,0 +1,86 @@
+require 'test_helper'
+
+class Kiba::Plus::Destination::MysqlTest < Minitest::Test
+  def before_setup
+    super
+
+    dest_mysql2_db = @@sequel_dbs[:mysql2_dest]
+
+    dest_mysql2_db.create_table! :customers do
+      primary_key :id
+      column :email, String
+      column :first_name, String
+      column :last_name, String
+    end
+
+    @dest_mysql2_db = dest_mysql2_db
+  end
+
+  def setup
+    @options = {
+      connect_url: @@connect_urls[:mysql2_dest],
+      prepare_sql: 'INSERT INTO customers VALUES (?, ?, ?, ?)',
+      columns: [:id, :email, :first_name, :last_name]
+    }
+
+    @obj = Kiba::Plus::Destination::Mysql.new(@options)
+  end
+
+  def test_initialize
+    assert_instance_of Mysql2::Client, @obj.client
+    assert_equal @options, @obj.options
+  end
+
+  def test_write_when_row_id_is_null
+    @obj.write({id: nil})
+
+    # TODO why eq 1 ?
```

Comment:
@hooopo Please check it, id is primary_key, when id is nil, insert it successful?